src/app/core/models/identify-user-payload.ts

```
@@ -0,0 +1,12 @@
+export interface IdentifyUserPayload {
+  id: string;
+  orgUserId: string;
+  orgId: string;
+  account: {
+    company_id: string;
+    region: string;
+  };
+  source: string;
+  is_admin: 'T' | 'F';
+  action_name: string;
+}
```

ESLint

[error] Interfaces should be defined only in files with .model.ts extension

(custom-rules/prefer-semantic-extension-name)